app/controllers/account_activations_controller.rb

```diff
@@ -13,11 +13,17 @@ def edit
     end
   end
 
+  private
+
   def load_user
     @user = User.find_by email: params[:email]
     return if @user
 
     flash[:danger] = t "user_not_found"
     redirect_to root_url
   end
+
+  def vaild_user
+    @user && !@user.activated? && @user.authenticated?(:activation, params[:id])
```

Comment:
```suggestion
    !@user.activated? && @user.authenticated?(:activation, params[:id])
```
mà ri thì cần j tách method

Reply:
@vanvtt-0952  rubocop nó bảo dài quá nên em tách ạ, em xuống dòng mãi mà nó không được